bench/gemm-benchmark.h

```diff
@@ -397,7 +397,7 @@ void GEMMBenchmark(benchmark::State& state,
 
   const size_t mc = state.range(0);
   const size_t nc = state.range(1);
-  const size_t kc = state.range(2);
+  const size_t kc = round_up_po2(state.range(2), 2 * kr * sr);
```

Comment:
This is ok. And if we round this up to bl we may not have to skip. But we may introduce redundancy across tests i.e. kc + 1 * (2srkr) and kc + 2 * (2srkr) where bl = 2 * (2srkr).